frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -297,19 +296,21 @@ export default memo(function HeatMap({
   // update displayedCellTypes and expandedTissues
   useEffect(() => {
     if (filteredCellTypes.length === 0) {
-      setDisplayedCellTypes(initialDisplayedCellTypes);
```

Comment:
no longer needs this, since we add `key={selectedOrganismId}` to the Heatmap, so it will automatically clear all states when the `key` changes